server/src/lib/api.ts

```diff
@@ -86,7 +95,8 @@ export default class API {
       bodyParser.raw({ type: 'image/*' }),
       this.saveAvatar
     );
-
+    router.post('/user_client/avatarclip', this.saveAvatarClip);
```

Comment:
I'd recommend going with `avatar_clip` for consistency's sake

Reply:
done